docs/release-notes/release-notes-0.16.0.md

```diff
@@ -27,6 +27,9 @@ transaction](https://github.com/lightningnetwork/lnd/pull/6730).
   method to the RPC to allow subscribing to updates from any inflight payment.
   Similar to TrackPaymentV2, but for any inflight payment.
 
+* [Catch and throw and error](https://github.com/lightningnetwork/lnd/pull/6945)
```

Comment:
nit: s/catch and throw and/catch and throw an/?

Reply:
hehehe yes, thanks :)

Reply:
updated!